website/content/docs/agent-and-proxy/agent/caching/persistent-caches/kubernetes.mdx

```diff
@@ -4,7 +4,7 @@ page_title: Kubernetes - Vault Agent Persistent Cache
 description: Kubernetes Persistent Cache for Vault Agent Caching
 ---
 
-# Vault agent kubernetes persistent cache
+# Vault Agent kubernetes persistent cache
```

Comment:
"kubernetes" should be init cap eg "Kubernetes"

Reply:
Good call! Updating